### PR TITLE
Add fragments for many more 061 products

### PR DESCRIPTION
**Pull request by pholleway**
**Related Issue(s):** src/stactools/modis/fragment update.

**Description:** Added files and folders needed for all datasets. Content not updated/added yet. 

**PR checklist:**

- [x] Code is formatted (run `scripts/format`).
- [x] Code lints properly (run `scripts/lint`).
- [x] Tests pass (run `scripts/test`).
- [ ] Documentation has been updated to reflect changes, if applicable.
- [x] Changes are added to the [CHANGELOG](../CHANGELOG.md).

### CODE REVIEW

**Review by gadomski — CHANGES_REQUESTED** [review]
Fragments look good. Can you add some example metadata XML files to `tests/data-files` as well? Thanks!

**Comment by pholleway** [comment]
> Fragments look good. Can you add some example metadata XML files to `tests/data-files` as well? Thanks!

Added v061 metadata xml for MOD17A2H and MOD17A3HGF. Do you we want to keep v006 metadata xml files?

**Comment by gadomski** [comment]
> Do you we want to keep v006 metadata xml files?

Yes please, other folks may be using the v6.0 STAC Items and Collections.

**Review by gadomski — CHANGES_REQUESTED** [review]
If we're going to include dummy fragments for the other products as part of this PR (which is 🆗 by me), let's clean up the dangling newlines and other weird stuff in the ones that were copied over from v6.0. For an example, check out src/stactools/modis/fragments/MOD10A1/061/bands.json.

**Comment by pholleway** [comment]
> If we're going to include dummy fragments for the other products as part of this PR (which is 🆗 by me), let's clean up the dangling newlines and other weird stuff in the ones that were copied over from v6.0. For an example, check out src/stactools/modis/fragments/MOD10A1/061/bands.json.

Will do! Thanks.

**Comment by pholleway** [comment]
> These files still look like they have `\n` scattered in the descriptions:
> 
> * src/stactools/modis/fragments/MOD10A1/061/collection.json
> * src/stactools/modis/fragments/MOD11A1/061/collection.json
> * src/stactools/modis/fragments/MOD11A2/061/collection.json
> * src/stactools/modis/fragments/MOD13A1/061/collection.json
> * src/stactools/modis/fragments/MOD13Q1/061/collection.json
> * src/stactools/modis/fragments/MOD14A1/061/collection.json
> * src/stactools/modis/fragments/MOD14A2/061/collection.json
> * src/stactools/modis/fragments/MOD15A2H/061/collection.json
> * src/stactools/modis/fragments/MOD16A3GF/061/collection.json
> * src/stactools/modis/fragments/MOD21A2/061/collection.json
> * src/stactools/modis/fragments/MOD21A2/061/item-properties.json

I only updated the "bands.json" the last round. About to commit the "collection.json" for MOD-. These will have the description updated.

**Review by gadomski — APPROVED** [review]

File: src/stactools/modis/fragments/MYD10A2/061/bands.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Maximum_Snow_Extent",
+        "description": "Maximum snow extent observed over an eight-day period."
+    },
+    {
+        "name": "Eight_Day _Snow_Cover",
+        "description": "r Snow chronology bit flags for each day in the eight-day observation period."
```
Comment (gadomski) [inline src/stactools/modis/fragments/MYD10A2/061/bands.json:8]: Is this leading `r` expected?

**Review by gadomski — APPROVED** [review]

**Review by gadomski — CHANGES_REQUESTED** [review]
src/stactools/modis/fragments/MCD15A3H/061/collection.json is empty

**Comment by pholleway** [comment]
> 



> src/stactools/modis/fragments/MCD15A3H/061/collection.json is empty

The MCD- "bands.json" and "collection.json" are in the other branch "Update v061 Fragment Files".

**Comment by gadomski** [comment]
Got it. In that case, I'll just merge that branch into here and close that PR. We'll do it all in one.

**Comment by pholleway** [comment]
> Got it. In that case, I'll just merge that branch into here and close that PR. We'll do it all in one.

Okay. Great. Thank you! Glad to have 1 branch now.

**Review by gadomski — APPROVED** [review]

File: src/stactools/modis/fragments/MOD10A1/061/raster-bands.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        "data_type": "uint8",
+        "unit": null,
+        "scale": null
+    },
+    "NSDI_Snow_Cover_Basic_QA": {
+        "data_type": "uint8",
+        "unit": null,
+        "scale": null
+    },
+    "NDSI_Snow_Cover _Algorithm_Flags_QA": {
```
Comment (gadomski) [inline src/stactools/modis/fragments/MOD10A1/061/raster-bands.json:12]:
```suggestion
    "NDSI_Snow_Cover_Algorithm_Flags_QA": {
```

File: src/stactools/modis/fragments/MOD10A2/061/raster-bands.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "Maximum_Snow_Extent": {
+        "data_type": "uint8",
+        "unit": null,
+        "scale": null
+    },
+    "Eight_Day _Snow_Cover": {
```
Comment (gadomski) [inline src/stactools/modis/fragments/MOD10A2/061/raster-bands.json:7]:
```suggestion
    "Eight_Day_Snow_Cover": {
```

File: src/stactools/modis/fragments/MOD10A2/061/bands.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Maximum_Snow_Extent",
+        "description": "Maximum snow extent over the 8-day period."
+    },
+    {
+        "name": "Eight_Day _Snow_Cover",
```
Comment (gadomski) [inline src/stactools/modis/fragments/MOD10A2/061/bands.json:7]:
```suggestion
        "name": "Eight_Day_Snow_Cover",
```

**Review by gadomski — CHANGES_REQUESTED** [review]
Sorry I didn't catch this sooner, but we can't have any spaces in the band names -- we use the band names to create the COG files, and so spaces would be inconvenient in the file names. Can you change any spaces to `_` (since underscore seems to be the convention for MODIS?

There are also some spurious whitespace in a couple of the NSIDC band names, I suggested changes on those.

File: src/stactools/modis/fragments/MOD10A2/061/bands.json (hunk 4 above)
Comment (pholleway) [reply src/stactools/modis/fragments/MOD10A2/061/bands.json:7]: Removed space in name for "bands.json" on MOD10A2, MOD13A1, MOD13Q1, MYD10A2, MYD13A1, and MYD13Q1.

File: src/stactools/modis/fragments/MOD10A2/061/raster-bands.json (hunk 3 above)
Comment (pholleway) [reply src/stactools/modis/fragments/MOD10A2/061/raster-bands.json:7]: Removed space in name for "raster-bands.json" in MCD64A1, MOD10A1, MOD10A2, MOD13A1, MOD13Q1, MYD10A1, MYD10A2, MYD13A1, and MYD13Q1.

File: src/stactools/modis/fragments/MOD10A1/061/raster-bands.json (hunk 2 above)
Comment (pholleway) [reply src/stactools/modis/fragments/MOD10A1/061/raster-bands.json:12]: Removed space in name for "raster-bands.json" in MCD64A1, MOD10A1, MOD10A2, MOD13A1, MOD13Q1, MYD10A1, MYD10A2, MYD13A1, and MYD13Q1.

**Comment by pholleway** [comment]
> Sorry I didn't catch this sooner, but we can't have any spaces in the band names -- we use the band names to create the COG files, and so spaces would be inconvenient in the file names. Can you change any spaces to `_` (since underscore seems to be the convention for MODIS?
> 
> There are also some spurious whitespace in a couple of the NSIDC band names, I suggested changes on those.

No problem! I've removed the spaces in the "bands.json" and "raster-bands.json" for MCD, MOD, and MYD-

**Review by gadomski — APPROVED** [review]

**Review by gadomski — APPROVED** [review]

**Pull request merged**